doc/plugins/traffic-split.md

```diff
@@ -111,6 +113,40 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
+2. Binding upstream services through the `upstream_id` attribute in the plugin.
```

Comment:
Don't use the term "services" here, people might be confused it with the APISIX Service.

Reply:
updated.